README.md

```diff
@@ -29,6 +29,7 @@ return retry(function (options) {
   ],
   backoffBase: 1000 // Initial backoff duration in ms. Default: 100,
   backoffExponent: 1.5 // Exponent to increase backoff each try. Default: 1.1
+  backoffJitter: 2500 // Amount of randomized jitter in ms to add to retry interval to spread retries out over time. Default: 0.0.
```

Comment:
Perhaps `150` is an example that fits better with the `backoffBase` example, if a user blindly copy pastes these examples the first two retries **could** be with 0 delay